test/unit/reports/get-invoice-number.test.js

```diff
@@ -0,0 +1,25 @@
+const extractedEvent = require('../../mocks/events/extracted')
+const enrichedEvent = require('../../mocks/events/enriched')
+
+const { UNKNOWN } = require('../../../app/constants/unknown')
+
```

Comment:
Remove space here?

Reply:
Suggest keeping to be consistent.

The approach I've taken is to group:
- mock pairs
- mocked objects
- constants
- target for test
- lets

Reply:
Yep, I would agree to keeping things consistent, so that's fine :)